### PR TITLE
Update presto images

### PR DESCRIPTION
**Pull request by yangzb09**
Update presto image from 
https://hub.docker.com/repository/docker/dashbase/presto/general
Built from https://github.com/dashbase/dashbase/tree/master/docker/presto

### CODE REVIEW

File: dashbase/values.yaml [hunk 1]
```diff
@@ -73,8 +73,9 @@ presto:
   enabled: false
   presto:
     image:
-      repository: "rluiarch/presto-wiwdata"
-      tag: "0.218-16"
+      repository: "dashbase/presto"
+      tag: "nightly"
+      pullPolicy: "Always"
```
Comment (sincejune) [inline dashbase/values.yaml]: I'm wondering if we need to add `Always` into users' cluster.

Comment (yangzb09) [reply dashbase/values.yaml]: If not nightly can't be the latest version. But I get your point.

Comment (sincejune) [reply dashbase/values.yaml]: Could we use `IfNotPresent` here? If a user wants to use it, he should specify the tag.

Comment (yangzb09) [reply dashbase/values.yaml]: How about using https://github.com/dashbase/dashbase-k8s/blob/0fda8ca0d76ece0367fd6d7ba327e2e79958f49a/presto/Chart.yaml#L2
as our dashbase/presto tag, set pullPolicy "Ifnotpresent".

Comment (sincejune) [reply dashbase/values.yaml]: Currently, I think use `nightly` as the default value is fine.

Comment (sincejune) [reply dashbase/values.yaml]: Of cause, that would be better if we can integrate the `dashbase_version`.

Comment (yangzb09) [reply dashbase/values.yaml]: We use nigthly as our default version, set pullPolicy "IfNotPresent".

**Comment by sincejune** [comment]
Can you check why the CI failed?

**Pull request merged**